gpkit/model.py

```diff
@@ -176,7 +176,7 @@ def zero_lower_unbounded_variables(self):
                      if bound == "lower"}
             self.substitutions.update(zeros)
 
-    def solve(self, solver=None, verbosity=1, skipfailures=True,
+    def solve(self, solver=None, verbosity=1, skipfailures=False,
```

Comment:
Ha. It shouldn't have been True by default, since it has a nice error message explaining what it does that shows up when it's false and a failure occurs.